desktop_version/src/FileSystemUtils.cpp

```diff
@@ -561,42 +561,16 @@ bool FILESYSTEM_openDirectoryEnabled()
 	return !SDL_GetHintBoolean("SteamTenfoot", SDL_FALSE);
 }
 
-#ifdef _WIN32
 bool FILESYSTEM_openDirectory(const char *dname)
 {
-	ShellExecute(NULL, "open", dname, NULL, NULL, SW_SHOWMINIMIZED);
+	char url[MAX_PATH];
+	SDL_snprintf(url, sizeof(url), "file://%s", dname);
```

Comment:
Why is there only one slash? Shouldn't it be `file:///`?

I tried `xdg-open file://` (which should be what `SDL_OpenURL()` ultimately ends up doing on my system), it said `xdg-open: file 'file://' does not exist`; but `xdg-open file:///` worked.

Reply:
`FILESYSTEM_openDirectory` is called with a path. With your change, it would be `file:////home/leo60228/.local/share/VVVVVV/levels` on my system, which is one too many slashes.

Reply:
On Windows, `file://C:\Users\leo60228\Documents\VVVVVV` is *presumably* correct.

Reply:
Having duplicate slashes on *nix is completely fine (`xdg-open file:////home/` works for me). I'm not sure about how many slashes Windows accepts, though.

Reply:
I'm not sure what your point is, then. Two slashes in the format string seems to be correct in all cases, even if more might work.

Reply:
Well, I don't know if it's correct on Windows. But I guess that's already addressed by you yourself admitting this needs to be tested on other platforms.